src/core/edgeNetwork/injectProcessWarningsAndErrors.js

```diff
@@ -24,7 +24,10 @@ export default ({ logger }) => {
     // want to process.
 
     const warnings = response.getWarnings();
-    const errors = response.getErrors();
+    const errors = response.getErrors().filter(error => {
+      // Remove the opt-out error because we don't throw errors when the user is opted out.
+      return error.code !== "EXEG-0301-403";
+    });
```

Comment:
I don't think this change should be necessary, but we will see what Rares thinks about the error.  This error is returning from every call the set-consent with consent set to "out".